assets/xml/objects/object_hakach_objects.xml

```diff
@@ -0,0 +1,28 @@
+<Root>
+    <File Name="object_hakach_objects" Segment="6">
+        <DList Name="gBotwHoleTrap1Dlist" Offset="0x01B0"/>
+        <DList Name="gBotwHoleTrap2Dlist" Offset="0x03F0"/>
+        <DList Name="gBotwTumbLidDlist" Offset="0x06B0"/>
```

Comment:
I think this should probably be "CoffinLid". "Tomb" is a more general term for a burial chamber.